### PR TITLE
update project & notify on deploys [SATURN-1399]

### PR DESCRIPTION
**Pull request by zarsky-broad**
Updates project to current standards (yarn, node 12) and adds notifications to `#workbench-release`

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.4.js"
```
Comment (panentheos) [inline .yarnrc:5]: If we're switching to yarn, we should change the `npm install` in the circleci config.

**Review by panentheos — COMMENTED** [review]
Looks ok in general. Is there any way we can test/verify this?

**Comment by zarsky-broad** [comment]
@panentheos the verification I've done is that the node script works after a yarn install.

**Review by panentheos — APPROVED** [review]
Looks good.

**Pull request merged**